src/Netscript/NetscriptHelpers.tsx

```diff
@@ -296,17 +297,17 @@ function validateHGWOptions(ctx: NetscriptContext, opts: unknown): CompleteHGWOp
     return result;
   }
   if (typeof opts !== "object") {
-    throw errorMessage(ctx, `BasicHGWOptions must be an object if specified, was ${opts}`);
+    throw errorMessage(ctx, `BasicHGWOptions must be an object if specified, was ${String(opts)}`);
   }
   // Safe assertion since threadOrOption type has been narrowed to a non-null object
   const options = opts as Unknownify<CompleteHGWOptions>;
   result.stock = !!options.stock;
   result.additionalMsec = number(ctx, "opts.additionalMsec", options.additionalMsec ?? 0);
   if (result.additionalMsec < 0) {
-    throw errorMessage(ctx, `additionalMsec must be non-negative, got ${options.additionalMsec}`);
+    throw errorMessage(ctx, `additionalMsec must be non-negative, got ${String(options.additionalMsec)}`);
```

Comment:
Did you mean `result.additionalMsec`?

```suggestion
    throw errorMessage(ctx, `additionalMsec must be non-negative, got ${result.additionalMsec}`);
```